### PR TITLE
add .editorconfig

### PR DESCRIPTION
**Pull request by DirtyF**
fix #44

### CODE REVIEW

File: .editorconfig [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 2
+indent_style = space
+insert_final_newline = true
+trim_trailing_whitespace = true
+
+[*.md]
+trim_trailing_whitespace = false
```
Comment (parkr) [inline .editorconfig]: I think this should still be `true`. Why did you choose `false` here?

Comment (pathawks) [inline .editorconfig]: For MarkDown specifically, trailing spaces can differentiate a linebreak from a new paragraph.

Comment (parkr) [inline .editorconfig]: Can you give me an example? Perhaps I have never used this feature.

Comment (pathawks) [inline .editorconfig]:
```md
Line1
Still line1

Line1  
Line2

Paragraph 1

Paragraph 2
```

Comment (pathawks) [inline .editorconfig]: Notice the two spaces after `Line1` before `Line2`

Comment (parkr) [inline .editorconfig]: Oh, weird.

**Comment by zdroid** [comment]
Great to see this implemented.

**Pull request merged**